distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc

```diff
@@ -199,37 +198,64 @@ When searching with wildcards, do not include the punctuation at the beginning o
+*** *Search Terms*: The value for the property to use during comparison.
+*** *Sorting*: Sort results by relevance, distance, created time, modified time or effective time.
+*** *Sources*: Perform an enterprise search (the local ${ddf-catalog} and all federated sources) or search specific sources.
 +
-*Between*: Set a beginning and end time to search between.
-+
-*Relative*: Search records relative to the current time.
-
-[[_spatial_search_details]]Spatial Search Details:: Search by latitude/longitude, USNG/MGRS, or UTM using a line, polygon, point-radius, or bounding box. Spatial criteria can also be defined by entering a *Keyword* for a region, country, or city in the *Location* section of the query builder.
-
-[[_type_search_details]]Type Search Details:: Search for specific content types.
-
-* *Advanced*: Advanced query builder to create more specific searches.
+** [[_advanced_query_builder_comparators]]*Advanced Query Builder Comparators*
+[no-bullet]
+*** *Textual*:
+[no-bullet]
+**** *CONTAINS*: Equivalent to <<_text_search_details,Basic Text Search>> with Matchcase set to No.
+**** *MATCHCASE*: Equivalent to <<_text_search_details,Basic Text Search>> with Matchcase set to Yes.
+**** *=*: Matches if an attribute is precisely equal to that search term.
+**** *NEAR*: Performs a fuzzy proximity-based textual search. A NEAR query of `"car street" within 3` will match a sample text of `the blue car drove down the street with the red building` because performing three word deletions in that phrase (`drove`, `down`, `the`) causes `car` and `street` to become adjacent.
 +
-[[_advanced_query_builder_details]]Advanced Query Builder Details:: *Operator*: If 'AND' is used, all the filters in the branch have to be true for this branch to be true. If 'OR' is used, only one of the filters in this branch has to be true for this branch to be true.
+More generally, a NEAR query of `"A B" within N` matches a text document if you can perform at most N insertions/deletions to your document and end up with `A` followed by `B`.
 +
-*Property*: Property to compare against.
+It is worth noting that `"street car" within 3` will not match the above sample text because it is not possible to match the phrase `"street car"` after only three insertions/deletions. `"street car" within 5` will match, though, as you can perform three word deletions to get `"car street"`, one deletion of one of the two words, and one insertion on the other side.
 +
-*Comparison*: How to compare the value for this property against the provided value.
+If multiple terms are used in the phrase, then the `within` amount specifies the total number of edits that can be made to attempt to make the full phrase match. `"car down street" within 2` will match the above text because it takes two word deletions (`drove`, `the`) to turn the phrase `car drove down the street` into `car down street`.
```

Comment:
Again, consider replacing _above text_ with the actual string, `the blue car drove down the street with the red building`